spec/state_factory_spec.rb

```diff
@@ -0,0 +1,57 @@
+RSpec.describe StateFactory do
+  CREATE_ACCOUNT_COMMAND = 'create'.freeze
+  LOAD_ACCOUNT_COMMAND = 'load'.freeze
+  DELETE_ACCOUNT_COMMAND = 'DA'.freeze
+  SHOW_CARDS_COMMAND = 'SC'.freeze
+  CREATE_CARD_COMMAND = 'CC'.freeze
+  DELETE_CARD_COMMAND = 'DC'.freeze
+  PUT_MONEY_COMMAND = 'PM'.freeze
+  WITHDRAW_MONEY_COMMAND = 'WM'.freeze
+  SEND_MONEY_COMMAND = 'SM'.freeze
+  EXIT = 'exit'.freeze
+
+  let(:current_subject) { described_class.new }
```

Comment:
This isn't necessary as of RSpec provides implicit subject. RSpec defines for each example subject variable containing exactly described_class.new